File: lab1_q1.c
#include<stdio.h>

int main()
{
    // declaring n as integer
    int n;
    //user input
    printf("Enter the natural number upto which you want to find the sum of all its previous numbers: ");
    scanf("%d",&n);
    printf("\n");
    // initialising sum and index
    int sum=0;
    int index=1;
    //checking boundary conditions 
    if(n<1){
        printf("The given input is not a natural number.");
    }
    else{
        // calculating sum in for loop
        while (index<=n){
            sum=sum+index;
            ++index;
        }
        // displaying output
        printf("Sum of first %d natural numbers: %d\n",n,sum);
    }
    printf("\n\n");
    return 0;
}

File: lab1_q1.py
#user input
n= int(input('Enter a natural number upto which you want to get the sum of all its previous numbers: '))
#initialising sum and index
sum=0
index=1
#checking boundary conditions 
if n<1:
      print('\nThe given input is not a natural number!')
else:
      #using while loop to find sum
      while index <= n:
            sum=sum+index
            index+=1
      #displaying output
      print('\nSum of first',n,'natural numbers:',sum)
print('\n\n')